README.md

```diff
@@ -77,22 +89,132 @@ In the sub-folders for plots that intersected with the orthomosaic, there will b
+``` 
+
+In step 6 we create a local folder to hold the output from processing:
+```bash
+mkdir -p /output
+```
+
+In step 7 we run the workflow to generate the orothomosaic image using ODM (OrthoDroneMap) and calculate plot-level canopy cover:
+```bash
+docker run --rm -v /var/run/docker.sock:/var/run/docker.sock -v /inputs:/scif/data/odm/images -v scif_output:/output -e INPUT_VOLUME=my_input -e OUTPUT_VOLUME=my_output -e "INPUT_IMAGE_FOLDER=/images" -e "OUTPUT_FOLDER=/output" agdrone/canopycover-workflow:latest run odm_workflow plot_shapes.shp my_input my_output
+```
+and we wait until it's finished.
+
+In step 8 we copy the results off the named output volume to our local folder:
+```bash
+docker run --rm -v /output:/results -v my_output:/output --entrypoint bash agdrone/canopycover-workflow:latest -c 'cp -r /output/* /results/'
+```
+The results of the processing are now in the `/output` folder.
+
+Finally, in step 9 we clean up the named volumes by deleting everything on them:
+```bash
+docker run --rm -v my_input:/input -v my_output:/output --entrypoint bash agdrone/canopycover-workflow:latest -c 'rm -r /input/* && rm -r /output/*'
+```
+
 ### Clean
+
 By executing the [scif](#scif) app named `clean` it's possible to clean up the output folder and other generated files.
 It's recommended, but not necessary, to run the clean app between processing runs by either running this command or through other means.
 
 **Example:**
 
-This docker command line will clean up the output files generated using the [Canopy Cover: Orthomosaic and Shapefile](#can_om_shp) example above.
+This docker command line will clean up the output files generated using the [Canopy Cover: Orthomosaic and Shapefile](#om_can_shp) example above.
 ```bash
-docker run --rm -v ~/inputs:/scif/data/odm/images -v ~/outputs:/scif/data/soilmask agdrone/canopycover-shape-workflow:latest run clean
+docker run --rm -v /inputs:/scif/data/odm/images -v /outputs:/scif/data/soilmask agdrone/canopycover-shape-workflow:latest run clean
 ```
 
 ## Build the container
+
 This section describes how the Docker container could be built.
 Please refer to the [Docker](https://www.docker.com/) documentation for more information on building Docker containers.
 
 ```bash
 cp jx-args.json.example jx-args.json
 docker build --progress=plain -t agdrone/canopycover-shape-workflow:latest .
 ```
+
+## Docker Sibling Containers
+
+Sibling containers is a technique for having one Docker container start another Docker container to perform some work.
+There are a variety of instances where using sibling containers can be desirable, but typically it's used when there's an existing Docker image available and a determination has been made that using other approaches is not desirable or, perhaps, possible.
+
+The following links provide additional information on sibling containers:
+* https://medium.com/@andreacolangelo/sibling-docker-container-2e664858f87a
```

Comment:
Tiny nitpick: Add `<` and `>` around links in Markdown.

Reply:
Done